### PR TITLE
add class 10 homework on model evaluation

### PR DESCRIPTION
**Pull request by cs79**
My folder now includes .py script for completing the Class 10 homework assignment on model evaluation.

### CODE REVIEW

File: alexlee/class10_hw_alexlee.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+## Part 1
+
+# get column names from this site:
+# http://archive.ics.uci.edu/ml/datasets/Glass+Identification
+cols = ['ID', 'RI', 'Na', 'Mg', 'Al', 'Si', 'K', 'Ca', 'Ba', 'Fe', 'Type']
+data_url = 'http://archive.ics.uci.edu/ml/machine-learning-databases/glass/glass.data'
+data = pd.read_table(data_url, sep=',', names=cols)
+
+# create a new column, "binary", that maps the Type column into 2 bins:
+data['binary'] = [0 if data['Type'][x] in {1, 2, 3, 4} else 1 for x in range(len(data))]
```
Comment (sinanuozdemir) [inline alexlee/class10_hw_alexlee.py]: good use of comprehensions!

File: alexlee/class10_hw_alexlee.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+print(metrics.accuracy_score(y_test, preds))            # check accuracy (88.89%)
+
+print(metrics.roc_auc_score(y_test, preds))              # check AUC (83.33%)
+
+## Part 4
+
+# train first model (logistic regression):
+mod1 = LogisticRegression()
+scores1 = cross_val_score(mod1, X, y, cv=5, scoring='roc_auc')
```
Comment (sinanuozdemir) [inline alexlee/class10_hw_alexlee.py]: You could also try excluding some predictors if they do not perform well.

**Comment by sinanuozdemir** [comment]
You did really well on this assignment! You clearly have a good grasp on this material.

File: alexlee/class10_hw_alexlee.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+from sklearn.linear_model import LogisticRegression     # import the estimator we want
+logreg = LogisticRegression()                             # instantiate estimator
+logreg.fit(X_train, y_train)                              # fit with training data
+preds = logreg.predict(X_test)                            # predict for test data
+
+print(ConfusionMatrix(list(y_test), list(preds)))       # see how we did
+
+print(metrics.accuracy_score(y_test, preds))            # check accuracy (88.89%)
+
+print(metrics.roc_auc_score(y_test, preds))              # check AUC (83.33%)
```
Comment (justmarkham) [inline alexlee/class10_hw_alexlee.py]: @cs79 As Sinan said, awesome job on this assignment! However, there is one very subtle error in this line of code... think for a minute about how AUC is calculated and whether or not you are giving this function what it is expecting... I bet you'll figure it out :)

Comment (cs79) [inline alexlee/class10_hw_alexlee.py]: Ah whoops, thanks for pointing that out.  I'm assuming in the lower code where the cross_val_score function is being asked to produce values for scoring method "roc_auc", it calculates the probabilities automatically?

Comment (justmarkham) [inline alexlee/class10_hw_alexlee.py]: It does indeed!

**Pull request merged**